src/libaktualizr/storage/invstorage.h

```diff
@@ -150,6 +161,9 @@ class INvStorage {
   virtual bool loadEcuSerials(EcuSerials* serials) = 0;
```

Comment:
Since we are extending the ecu_serials table with the additional attributes/columns why not extend the corresponding write/read functions, i.e loadEcuSerials/storeEcuSerials/clearEcuSerials instead of introducing the new methods.

Reply:
To answer generally, the fact that this is a column shouldn't really drive the storage api changes, in my opinion. I see it as two independent choices: what makes the api more usable and where does it make sense to store this additional data.

Including it in the existing functions should be possible but I'm afraid it would require a lot of changes in different places, for example the tests.

Reply:
At the moment, I think, that it would be better not to touch this table `ecu_serials` nor the API, just introduce another table.

Reply:
Do you imply that your original suggestion to add these to load/store/clear EcuSerials is now obsolete?

Reply:
> Do you imply that your original suggestion to add these to load/store/clear EcuSerials is now obsolete?

Yes, I am pretty sure that it's wrong direction now.